code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -585,6 +662,10 @@ var/global/list/robot_modules = list(
 	name = "research module"
 	channels = list("Science" = 1)
 	sprites = list(
+					"L'Ouef" = "peaceborg",

```

Comment:
look I know I have a french name but don't name my beloved panda eggs in french ;_;